terraform/trial_rg/main.tf

```diff
@@ -115,15 +116,31 @@ resource "random_password" "roles_sql_password" {
 
 # Create a roles and permissions SQL
 # Deploy a sql server and db for fhir before we create the web app
-module "fhir_sql_server" {
+module "roles_sql_server" {
```

Comment:
why has fhir_sql_server been renamed to roles_sql_server ? is there a distinction between the FHIR and data sql server instances ?

Reply:
I have created 2 sql servers. one for fhir and one for roles